### PR TITLE
turn off normalize_ir, turn on use_functionalization by default

### PR DESCRIPTION
**Pull request by bdhirsh**
fresh PR without ghstack. Before landing, I still need to

(1) rebase on top of https://github.com/pytorch/torchdynamo/pull/1048
(2) mark the 6 new failing models as skips (and create issues tracking them)

### CODE REVIEW

**Review by Chillee — APPROVED** [review]

**Comment by desertfire** [comment]
If you are talking about marking 6 new failing model to the CI skip list, you can wait for my https://github.com/pytorch/torchdynamo/pull/1040 to land. Also did you test on hf and timm as well?

**Review by ezyang — APPROVED** [review]

**Comment by bdhirsh** [comment]
@desertfire thanks, I'll wait for your PR to land and add to the skip list.

> Also did you test on hf and timm as well?

Yep - I ran the hf and timm benchmarks locally with the before and after, as a sanity check that functionalization isn't regressing a large number of models. I'm not 100% confident in my results (a few models didn't run at all for some reason), but I counted a total of 8 new failures from HF, and 4 new failures from timm.

Once this PR lands I'll work on triaging the failures and making issues for them.

File: .circleci/config.yml
Comment (bdhirsh) [inline .circleci/config.yml:72]: hmm @desertfire I'm not sure why your changes are showing up in my PR - it looks like https://github.com/pytorch/torchdynamo/pull/1040/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47 already landed.

Comment (desertfire) [reply .circleci/config.yml:72]: Update main and then `git rebase`?

**Comment by bdhirsh** [comment]
looks like CI is getting rate limited

**Pull request merged**
